src/collections/utils.ts

```diff
@@ -75,18 +74,17 @@ function getBaseAPIPath(pathFromUrl?: string | null) {
  * @returns URL as string
  */
 export function createAstraUri (
-    databaseId: string,
-    region: string,
-    keyspace: string,
+    apiEndPoint: string,
+    keyspace?: string,
     applicationToken?: string,
     baseApiPath?: string,
     logLevel?: string,
     authHeaderName?: string,
 ) {
-    const uri = new url.URL(`https://${databaseId}-${region}.apps.astra.datastax.com`);
+    const uri = new url.URL(apiEndPoint);
```

Comment:
We should probably consider a case when the full URL (including `/api/json/v1`) is provided. Maybe append only if the provided endpoint ends with `apps.astra.datastax.com`.

Reply:
I think probably no.
User won't get any information about '/api/json/v1' from the new portal UI.
Also, other jsonapi clients are not doing this, I think.